gcovr/__main__.py

```diff
@@ -312,7 +312,6 @@ def collect_coverage_from_tracefiles(covdata, options, logger):
             for trace_file in trace_files:
                 datafiles.add(normpath(trace_file))
 
-    options.root_dir = os.path.abspath(options.root)
```

Comment:
Is this safe to remove? Do other parts of the code rely on the `root_dir` being absolute?

Reply:
The same thing is done in the main function (line230).